README.adoc

```diff
@@ -0,0 +1,274 @@
+= Atomic Developer Bundle
+:toc:
+
```

Comment:
>  I already mentioned this when I submitted the PR: the problem with the TOC being live-rendered here is apparently unrelated to the TOC in VSM, 

How is it unrelated? Apparently TOC can be rendered on GitHub as one can see at vagrant-service-manager. So why can we not get them to render here? I think missing them on the rendered GitHub pages is a big loss.

> because the version of asciidoc changed and GitHub doesn't render TOCs anymore. 

So why does it render on vagrant-service-manager. I would love to understand this one better.

> I already made a comment about this and @bexelbie agreed that we shouldn't invest in it (I did a bunch of research and it looks like there are some really ugly hacks but no out-of-the-box way to do it now). 

So what are the hacks? Are there any GitHub internal issues somewhere?